### DISCUSSION TITLE
Could not poll the database: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed

### DISCUSSION THREAD

**Discussion by adman120** [firefly-iii/firefly-iii#5917] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I keep getting the following when trying to launch
Could not poll the database: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known (SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

### Expected behaviour

Not to happen

### Debug information

Logs attache
[logs.txt](https://github.com/firefly-iii/firefly-iii/files/8311954/logs.txt)
d

**Comment by adman120** [comment]
Running from docker

**Comment by JC5** [comment]
```
Error when connecting to DB: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known
```

The database isn't running or the configuration isn't correct. Since this doesn't seem to be a bug, I've turned this into a discussion.

**Reply by adman120** [reply]
ok, im confused because I know the database in running and my .env file points directly to it.
[.firefly_env.txt](https://github.com/firefly-iii/firefly-iii/files/8311974/default.firefly_env.txt) 
and I can connect to it in PHP manager.

**Reply by JC5** [reply]
There is a space in your env file. I'm not sure if those are trimmed automatically.

```
DB_HOST= 192.168.91.28
```

**Reply by adman120** [reply]
Ok, that solved one issue, now this happens Could not poll the database: SQLSTATE[HY000] [1045] Access denied for user 'firefly'@'docker_firelyapp_1.t2_proxy' (using password: YES) (SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

**Reply by JC5** [reply]
The error seems obvious enough.
